lib/board.rb

```diff
@@ -0,0 +1,9 @@
+module Board
+  def display_board
+    puts " #{@board[0]} | #{@board[1]} | #{@board[2]} "
+    puts ' ----------- '
+    puts " #{@board[3]} | #{@board[4]} | #{@board[5]} "
+    puts ' ----------- '
+    puts " #{@board[6]} | #{@board[7]} | #{@board[8]} "
+  end
```

Comment:
## Remove puts outside main.rb file
Any puts staments outside main rb it; not good practice, therefore we have only reutns a cncatendes string, follow this example

```ruby
def display_board
  boardString='first-line'
  boardString+='second-line'
  ...
  boardString

end
```